capstone/src/main/java/com/google/sps/servlets/CreateNewUserServlet.java

```diff
@@ -43,28 +45,20 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     UserService userService = UserServiceFactory.getUserService();
     if (userService.isUserLoggedIn()) {
       DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-      datastore.put(createUserEntity(userService, first, last, phone, interests));
+      String userId = userService.getCurrentUser().getUserId();
+      User user = new User(/* userId */ userId,
+                          /* firstName */ first,
+                          /* lastName */ last, 
+                          /* email */ userService.getCurrentUser().getEmail(),
+                          /* phoneNumber */ phone, 
+                          /* profilePic */ "", 
+                          /* badges */ new LinkedHashSet<Badge>(), 
+                          /* groups */ new LinkedHashSet<Long>(), 
+                          /* interests */ interests);
```

Comment:
```java
User user = new User(
  userId,
  firstName,
  lastName, 
  /* email= */ userService.getCurrentUser().getEmail(),
  phoneNumber, 
  /* profilePic= */ "", 
  /* badges= */ new LinkedHashSet<Badge>(), 
  /* groups= */ new LinkedHashSet<Long>(),
  interests
);
```

Reply:
You don't need a comment for a parameter if the parameter and the variable share the same exact name